jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -1169,6 +1175,13 @@ public void parse(final String rawarg, String source)
             return;
         }
 
+        if (arg.startsWith("--pid-file="))
+        {
+            setProperty("jetty.pid.file", pidFile, source);
+            xmlRefs.add("etc/jetty-pid.xml");
```

Comment:
I don't know if this is the right way, as `jetty-started.xml` is similar but done in a completely different way.

Reply:
It is similar, yes.
I'll see if I can work this into jetty.sh exclusively (with no changes to StartArgs)